app/src/main/java/org/akvo/flow/activity/BackActivity.java

```diff
@@ -19,21 +19,21 @@
 
 package org.akvo.flow.activity;
 
-import android.os.Bundle;
 import android.support.v7.app.ActionBar;
 import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
 import android.view.MenuItem;
 
-public abstract class BackActivity extends AppCompatActivity {
+import org.akvo.flow.R;
 
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
+public abstract class BackActivity extends AppCompatActivity {
```

Comment:
This class contains the back functionality, but the base activity is for injection, they will not be merged, maybe this one will extend BaseActivity but I am not sure yet.